### DISCUSSION TITLE
JavaScript Primerの次にやったこと

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1306] [体験のシェア]
JavaScript Primerを読んだあとに読んだもの、やったことなどの体験を書いていくスレッドです。

**Comment by yumetodo** [comment]
後というより同時並行で
https://developer.mozilla.org/ja/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript
をやりました。

**Comment by yo-goto** [comment]
「JavaScript Primer」にものすごくお世話になっています。分かりやすい本を書いていただきありがとうございます!
Primer を読み終えた後に特に非同期処理についての理解を深めるために色々と試行錯誤した過程を zenn で次の記事で書かせていただきました(「Promise の本」にも大変お世話になりました)。稚拙な記事ですが、フィードバックや他の人の参考になるかもしれないのでもしよければ読んでみてください。
https://zenn.dev/estra/articles/js-async-programming-roadmap

具体的には JSConf などの動画や MDN, freecodecamp などの記事を中心に学習し、最終的には ECMAScript や HTML の仕様をつまみ食いで見ていく感じで進めました。
